VultisigApp/VultisigApp/Services/CryptoPriceService.swift

```diff
@@ -67,31 +262,38 @@ private extension CryptoPriceService {
         return ResolvedSources(providerIds: providerIds, contracts: contracts)
     }
     
-    func fetchPrices(ids: [String]) async throws {
-        let idsQuery = ids
-            .filter { !$0.isEmpty }
-            .joined(separator: ",")
+    func fetchPrices(ids: [String], coins: [Coin]) async throws {
+        let idString = ids.joined(separator: ",")
+        let endpoint = Endpoint.fetchCryptoPrices(ids: idString, currencies: "usd")
```

Comment:
coingecko is able to return different price in different currencies, 
This hardcoded it to USD , which is not necessary